src/index.js

```diff
@@ -73,14 +72,17 @@ class ReactTooltip extends Component {
       currentTarget: null, // Current target of mouse event
       ariaProps: parseAria(props), // aria- and role attributes
       afterUnfreeze: () => {},
-      isEmptyTip: false,
-      disable: false
+      disable: false,
+      originTooltip: null,
+      isMultiline: false
```

Comment:
From the docs:

|Global|Specific|Type|Values|Description|
|------|--------|-----|--------|----------|
|multiline|data-multiline|Bool|true, false|support &lt;br&gt;, &lt;br /&gt; to make multiline|